library/CHART-TEMPLATE/tests/common_images_pull_secrets_test.yaml

```diff
@@ -4,17 +4,18 @@ templates:
 release:
   name: my-release
   namespace: my-namespace
-set:
 tests:
   - it: does not set imagePullSecrets if not specified
     set:
+      cluster: test-cluster
```

Comment:
Why do we need to set `cluster: test-cluster` for _all_ test cases?

Reply:
removed